Fixes to labs

sharksforarms opened the pull request:
Currently in a ddev training, I'll be pushing fixes to examples here as I come across them.

Commit:
returns https now

squizzi commented on api-tests/tests/api-spec.js:
```diff
@@ -4,8 +4,8 @@ var base_url = "http://api:8080/";
     it("should respond with the URL of a cat GIF", function(done) {
         request(base_url + 'api/pet', function(error, response, body){
-            expect(body).toMatch(/\{"imageId":\d+,"url":"http:/);
+            expect(body).toMatch(/\{"imageId":\d+,"url":"https:/);
```
The better way to fix this would be to allow both `http` and `https` `toMatch` with something like:

```
            expect(body).toMatch(/\{"imageId":\d+,"url":"http(s?):/);
```

Commit:
there's one http case, so it could be http and not always https

database/docker-entrypoint-initdb.d/init-db.sql

INSERT INTO images (description, url) VALUES('cat image', 'http://placekitten.com/g/400/400');

katieannemills commented:
cool, thanks @sharksforarms and @squizzi - I should change the one and only `http` link to `https`, then this will always work~
